templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.error %}

```

Comment:
The condition `{% if form.error %}` is incorrect. The correct syntax should be `{% if form.errors %}` to check for any form validation errors.